File: Java/UberX.java
class UberX extends Car {
    String brand;
    String model;

    public UberX(String license, String driver, String brand, String model){
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }

    void printDataCarUberX() {
        if (passenger != null) {
            System.out.println("License: " + license + " Name Driver: " + driver + " Brand: " + brand + " Model: " + model + " Passengers: " + passenger);
        }
    }

    public Integer getPassenger(){
        return passenger;
    }
    public void setPassenger(Integer passenger){
        if (passenger == 4) {
            this.passenger = passenger;
        } else {
            System.out.println("Necesitas agregar 4 pasajeros");
        }
    }

}

File: PHP/uberX.php
<?php
require_once('car.php');
class UberX extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }
    public function PrintDataCar(){
        parent::PrintDataCar();
        echo" 
        Modelo: $this->model
        Marca: $this->brand 
        <br>";
    }
}